privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -338,17 +344,24 @@ def attack(self):
             [[1, 0]]-> property; [[0, 1]]-> negation property
         :rtype: np.ndarray with shape (1, 2)
         """
-
+        if(self.verbose>0):
+            print("Initiating Property Inference Attack ... ")
+            print("Extracting features from target model ... ")
         # extract features of target model
         feature_extraction_target_model = self.feature_extraction(self.target_model)
 
+        if(self.verbose>0):
+            print(feature_extraction_target_model.shape, " --- features extracted from the target model.")
+
         # set ratio and size for unbalanced data sets
         size_set = 1000
 
         # balanced ratio
         num_elements = int(round(size_set / 2))
         neg_property_num_elements_per_class = {0: num_elements, 1: num_elements}
 
+        if(self.verbose>0):
+            print("Creating set of balanced shadow classifiers ... ")
```

Comment:
A more concrete message would include the number of balanced shadow classifiers

Reply:
Good point. I changed that